tests/run-make/extern-fn-reachable/rmake.rs

```diff
@@ -0,0 +1,45 @@
+//! Smoke test to check that that symbols of `extern "C"` functions and `#[no_mangle]` rust
+//! functions:
+//!
+//! 1. Are externally visible in the dylib produced.
+//! 2. That the symbol visibility is orthogonal to the Rust nameres visibility of the functions
+//!    involved.
+
+//@ ignore-cross-compile
+
+use run_make_support::object::{self, Object};
```

Comment:
is `Object` used?

Reply:
Yes, `Object` the trait needs to be in scope for `Object::exports`.